src/Shared/EdmHelpers.cs

```diff
@@ -5,7 +5,7 @@
 using System.Globalization;
 using System.Linq;
 using Microsoft.OData.Edm.Library;
-using Microsoft.Restier.Core.Properties;
+using Microsoft.Restier.WebApi.Properties;
```

Comment:
This one I consider more changes, as it refer to a resource properties, and resource class is auto generated and it is internal, we can not really share the class, I was thinking just move it to WebApi project and no other class are using it now.
Comments?